src/js/localstorage.js

```diff
@@ -0,0 +1,187 @@
+//TESTING LOCALSTORAGE WHETHER IT WORKS OR EXIST 
```

Comment:
As I said in previous comment, we do not need to use comments at all. Instead name functions and variables so they will tel us what they do. This is crucial in software development.